lib/binary_to_decimal.rb

```diff
@@ -4,6 +4,12 @@
 # The least significant bit is at index 7.
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
-def binary_to_decimal(binary_array)
-  raise NotImplementedError
+
+array = Array.new(8) { rand(0..1) }
```

Comment:
You shouldn't need lines 8 and 15. Run rake and make sure that all the tests pass.